build.sbt

```diff
@@ -345,7 +345,8 @@ lazy val dist = {
       mappings in Universal ++= Seq(
         (crossTarget in generic).value / "genericModel.jar" -> "model/genericModel.jar",
         (crossTarget in flinkProcessManager).value / s"nussknacker-flink-manager.jar" -> "managers/nussknacker-flink-manager.jar",
-        (crossTarget in engineStandalone).value / s"nussknacker-standalone-manager.jar" -> "managers/nussknacker-standalone-manager.jar"
+        (crossTarget in engineStandalone).value / s"nussknacker-standalone-manager.jar" -> "managers/nussknacker-standalone-manager.jar",
+        (crossTarget in openapi).value / "openapi.jar" -> "model/openapi.jar"
```

Comment:
WDYT about some nested directory for additional component providers like `model/components` I see two benedfits:
- it will be clear that there are not models but just plugins with extra components
- in the futher we could add some discovery mechanizm that will check this directory and add all plugins to model's classpath

Reply:
Done